specificatie/openapi.yaml

```diff
@@ -17,6 +17,9 @@ info:
   license:
     name: European Union Public License, version 1.2 (EUPL-1.2)
     url: https://eupl.eu/1.2/nl/
+tags:
+  - name: Ingeschreven Personen
+    description: Zoeken en raadplegen van ingeschreven personen
```

Comment:
description moet veranderen in "Zoeken ingeschreven personen".

Reply:
@CathyDingemanse we willen toch naast het zoeken (POST) ook het raadplegen (GET met uuid) doen? Dan blijft het zoeken en raadplegen.